### DISCUSSION TITLE
Sorting in Front-end API?

### DISCUSSION THREAD

**Discussion by Kealtie** [swellstores/community#176] [Help] [answered]
Hi, I couldn't find any documentation about sorting products using the Swell for Nuxt client.
Could anyone help me out? I want to sort based on name ascending, name from a-z and price ascending.

Thanks in advance!

**Comment by rodrigoddalmeida** [comment, marked as answer]
You can sort products by name, price on our Frontend APIs with the following:

- Swell.js
```js
await swell.products.list({
  sort: "name asc, price asc",
});
```

- GraphQL
```gql
query {
  products(sort: "name asc, price asc") {
    results {
      ...
    }
  }
}
```

**Reply by Kealtie** [reply]
Hi @rodrigoddalmeida, Thanks for your reply. I tried using the Swell.js method, but this changes nothing to the response and the order of the products.

**Reply by rodrigoddalmeida** [reply]
@jordytenden did you try using a comma-separated string? (I've updated the example above as well)

```js
await swell.products.list({
  sort: "name asc, price asc",
});
```

It avoids any current issues with query string encoding.
